packages/builders/scripts/inter-protocol/update-vf-electorate.js

```diff
@@ -0,0 +1,24 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
```

Comment:
I'd be inclined to skip a builder script for this exercise.

Reply:
done